osu.Game/Configuration/SettingSourceAttribute.cs

```diff
@@ -20,22 +21,36 @@ namespace osu.Game.Configuration
     [AttributeUsage(AttributeTargets.Property)]
     public class SettingSourceAttribute : Attribute
     {
+        public enum OrderMode
+        {
+            FIRST,
+            ORDERED_RELATIVE,
+            UNORDERED,
+            LAST
```

Comment:
Enum members in C# are usually Pascal-cased (`First`, `OrderedRelative`...)

Reply:
Will note enum casing for future contributions